common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/validation/DaysSinceSymptomsRangeSpecification.java

```diff
@@ -0,0 +1,85 @@
+package app.coronawarn.server.common.persistence.domain.validation;
+
+import app.coronawarn.server.common.persistence.domain.validation.util.Range;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Function;
+
+public enum DaysSinceSymptomsRangeSpecification {
+
+  /**
+   * As of CWA version 1.5, the range below defines the range of DSOS values accepted by the
+   * Exposure Notification framework.
+   */
+  ExposureNotificationAcceptedRange(-14, 21, (dsos) -> 0),
```

Comment:
Please recheck this with Max. My understanding was that the range should be -14 ->14